report_page_orientation/views/views.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+<!-- abandoned great idea
+<template id="minimal_layout" inherit_id="web.minimal_layout">
+
+        <xpath expr="//script" position="after">
+            <script t-if="subst_multi">
+                    function subst_multi() {
+                    var vars = {};
+                        debugger;
+                        var x = document.location.search.substring(1).split('&amp;');
+                        for (var i in x) {
+                            var z = x[i].split('=', 2);
+                            vars[z[0]] = 66;
+                        }
+                        var x = ['sitepage', 'sitepages', 'section', 'subsection', 'subsubsection'];
+                        var z = {'sitepage': 'page', 'sitepages': 'topage'};
+                        for (var i in x) {
+                            var y = document.getElementsByClassName(z[x[i]] || x[i])
+                            for (var j=0; j&lt;y.length; ++j)
+                                y[j].textContent = 44;
+                        }
+
+                        var index = 76;
+                        var header = document.getElementById('minimal_layout_report_headers');
+                        if(header){
+                            var companyHeader = header.children[index];
+                            header.textContent = '';
+                            header.appendChild(companyHeader);
+                        }
+                        var footer = document.getElementById('minimal_layout_report_footers');
+                        if(footer){
+                            var companyFooter = footer.children[index];
+                            footer.textContent = '';
+                            footer.appendChild(companyFooter);
+                        }
+                    }
+            </script>
+        </xpath>
+        <xpath expr="//body" position="attributes">
+            <attribute name='t-att-onload'>subst and subst() ; subst_multi and subst_multi() </attribute>
+        </xpath>
+
+    </template>
+-->
+
+    <!-- context idea:
+        disable subst in footer wich means that page/topage span will be empty.
+        put right next to it 2 values passed via context.
+
+        DOES NOT WORK
+
+    <template id="minimal_layout" inherit_id="web.external_layout_standard">
+        <xpath expr="//span[@class='page']" position="after">
+            <span><t t-esc="env.context.get('num_page')" /></span>
+         </xpath>
+        <xpath expr="//span[@class='topage']" position="after">
+            <span><t t-esc="env.context.get('tot_page')"/></span>
+         </xpath>
+     </template>
+     -->
+</odoo>
```

Comment:
Can we remove Gio's abandoned great ideas?

Reply:
@thomaspaulb     a  "great idea" can be removed when a working idea exists.
removing this myself.